travispy/entities/repo.py

```diff
@@ -71,3 +71,38 @@ def last_build(self):
     def find_one(cls, session, entity_id, **kwargs):
         result = super(Repo, cls).find_one(session, entity_id, **kwargs)
         return result
+
+
+    def _set_hook(self, flag):
+        url = self._session.uri + '/hooks/{}'.format(self.id)
+        data = {
+            "hook": {
+                "active": flag
+            }
+        }
+
+        response = self._session.put(url, json = data)
+
+        return response.status_code == 200
```

Comment:
I think you can rewrite here a little bit:

    if response.status_code == 200:
        self.active = flag
        return True
    else:
        return False

This way all ``disable`` and `` enable``  code can be changed by

    def disable(self):
        return self._set_hook(False)

    def enable(self):
        return self._set_hook(True)